### PR TITLE
Improve display of Battery and GPS data on HUD

### PR DESCRIPTION
**Pull request by sborenstein-ift**
Changes to the HUD add display improvements for the Battery and GPS displays in single and dual line modes, and move all text to be "bottom justified".

### CODE REVIEW

File: ExtLibs/Controls/HUD.cs [hunk 1]
```diff
@@ -31,9 +31,9 @@
 
 namespace MissionPlanner.Controls
 {
-    public class HUD2: HUD
+    public class graphicsObject: HUD
```
Comment (meee1) [inline ExtLibs/Controls/HUD.cs:34]: can this be undone

File: ExtLibs/Controls/HUD.cs [hunk 2]
```diff
@@ -102,6 +102,21 @@ private void FreeBitmap()
                 bitmap = null;
             }
         }
+
```
Comment (meee1) [inline ExtLibs/Controls/HUD.cs:105]: and this undone

**Comment by meee1** [comment]
could you post a before and after screenshot?

**Comment by sborenstein-ift** [comment]
image
image
image

Attached are normal 2-line, min-size 2-line and single line screen caps.

**Comment by sborenstein-ift** [comment]
Here's the "before" image, single line and a quick battery 2-line. Note, in dual batt and dual GPS view, the lines have no space between. 
image
image

File: ExtLibs/Controls/HUD.cs (hunk 1 above)
Comment (sborenstein-ift) [reply ExtLibs/Controls/HUD.cs:34]: Good question. I did not explicitly do this. I wonder if MVS took over here.

**Comment by sborenstein-ift** [comment]
@meee1 I just reverted the first commit, which wasn't necessary, which addresses the questions you had. I think when I first started modifying this file, MVS did some wierd things to the RESX file. That wasn't necessary, I just removed that commit. Only the CS file is modified and changes should make more sense.

**Comment by meee1** [comment]
> @meee1 I just reverted the first commit, which wasn't necessary, which addresses the questions you had. I think when I first started modifying this file, MVS did some wierd things to the RESX file. That wasn't necessary, I just removed that commit. Only the CS file is modified and changes should make more sense.

yeah, i was going to say the resx is redundant, as its 100% user drawn in code, so no designed parts.

**Pull request merged**